opm/simulators/flow/partitionCells.cpp

```diff
@@ -40,68 +48,325 @@
 #include <numeric>
 #include <stdexcept>
 #include <string>
+#include <tuple>
 #include <type_traits>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+// ===========================================================================
+// Zoltan-based partitioning if availble in this build configuration.
```

Comment:
available

Reply:
Thanks for noticing. I've pushed an update to fix that.